pkg/config/config.go

```diff
@@ -87,6 +90,24 @@ func (c *Config) InitConfig() {
 		TimestampFormat: time.RFC1123,
 	}
 
+	log.SetFormatter(logFormatter)
+
+	// Set log level
+	switch c.LogLevel {
+	case "debug":
+		log.SetLevel(log.DebugLevel)
+	case "info":
+		log.SetLevel(log.InfoLevel)
+	case "trace":
+		log.SetLevel(log.TraceLevel)
+	case "warn":
+		log.SetLevel(log.WarnLevel)
+	case "error":
+		log.SetLevel(log.ErrorLevel)
+	default:
+		log.Fatalf("Unrecognized log level value: %s. Expected one of debug, info, warn, error.", c.LogLevel)
+	}
+
```

Comment:
this needs to be set as early as possible so that we can debug log in the remainder of this function